tools/gradle-plugins/extensions/markdown/src/main/kotlin/com/varabyte/kobwebx/gradle/markdown/KobwebxMarkdownPlugin.kt

```diff
@@ -28,10 +29,14 @@ class KobwebxMarkdownPlugin : Plugin<Project> {
         val convertTask = project.tasks
             .register<ConvertMarkdownTask>("kobwebxMarkdownConvert", kobwebBlock, markdownBlock)
 
+        val processTask = project.tasks
+            .register<ProcessMarkdownTask>("kobwebxMarkdownDataProcess", kobwebBlock, markdownBlock)
+
         project.buildTargets.withType<KotlinJsIrTarget>().configureEach {
             val jsTarget = JsTarget(this)
             project.kotlin.sourceSets.named(jsTarget.mainSourceSet) {
                 kotlin.srcDir(convertTask)
+                kotlin.srcDir(processTask)
```

Comment:
```suggestion
                kotlin.srcDir(processTask.map { it.getGenDir() })
```
To ensure that only the kt files are treated as sources. 
I'd also probably rename `getGenDir()` to `genGenSrcDir` or `getGenKtDir` just to be more explicit.

Reply:
That's a good idea.
Let's go with `genGenSrcDir`.